e2e_tests/test_workspace_templates.py

```diff
@@ -13,7 +13,9 @@
 
 
 workspace_templates = [
-    (strings.BASE_WORKSPACE)
+    (strings.BASE_WORKSPACE),
+    (strings.UNRESTRICTED_WORKSPACE),
+    (strings.AIRLOCK_IMPORT_REVIEW_WORKSPACE)
```

Comment:
What will this addition actually test?

Reply:
That these templates deploy. There are a few issues customers hitting issues:

https://github.com/microsoft/AzureTRE/issues/3700 etc.

Issue is the versions in their Dockerfile are not being updated when issues are fixed in the base workspace.